_posts/2017-01-06-Linux-on-Asus-ROG.md

```diff
@@ -12,36 +13,83 @@ You will be brought to a page where you can make some edits.
 3. add the following right after the word “splash”
     >nouveau.modeset=0 tpm_tis.interrupts=0 acpi_osi=Linux i915.preliminary_hw_support=1 idle=nomwait
 4. Press the “f10” key to boot into the installer
-5. When the desktop load move your mouse into the upper left hand corner, when the app launcher shows itself click on the very top icon “Install Ubuntu GNOME 16.04 LTS”
+5. When the desktop load move your mouse into the upper left hand corner, when the app launcher shows itself click on the very top icon “Install Ubuntu GNOME 18.04 LTS”
 6. Do not install third party software.
 7. Then follow the steps to install Ubuntu.
 8. After installation click “Keep trying Ubuntu”
 9. Shut down (NOT RESTART) your computer, remove installation media, hit enter.
 10. Hold down shift then press the power button to turn on your computer. (holding down shift while powering on will load the grub boot menu)
 11. When the grub boot menu appears press “e” and make the changes in step 3.
 12. Press “f10”
-13. Once your back into the desktop open your terminal and enter:
+13. Once your back into the desktop open your terminal. Press `Ctrl+Alt+T` if touch pad is not working and enter:
     >sudo gedit /etc/default/grub
 14. Find the line that reads ‘GRUB_CMDLINE_LINUX_DEFAULT=”quiet splash”‘ and change it to read
-    >‘GRUB_CMDLINE_LINUX_DEFAULT=”quiet splash nouveau.modeset=0 tpm_tis.interrupts=0 acpi_osi=Linux         i915.preliminary_hw_support=1 idle=nomwait”‘
-15. Save the changes then exit gedit.
-16. In your terminal run the command: sudo update-grub
-17. Reboot your computer: sudo reboot.
+    >‘GRUB_CMDLINE_LINUX_DEFAULT=”quiet splash nouveau.modeset=0 tpm_tis.interrupts=0 acpi_osi=Linux i915.preliminary_hw_support=1 idle=nomwait”‘
```

Comment:
Could you use code blocks instead using quote `>`.
This goes for all the following quotes.